### DISCUSSION TITLE
why i can't connet?

### DISCUSSION THREAD

**Discussion by Ting230** [Melledy/Grasscutter#2541] [Q&A] [answered]
17:57:47.792 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3998.lua
17:57:47.879 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3114.lua
17:57:47.881 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3115.lua
17:57:47.882 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3116.lua
17:57:47.883 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3501.lua
17:57:47.884 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3502.lua
17:57:47.885 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3503.lua
17:57:47.885 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3504.lua
17:57:47.886 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3505.lua
17:57:47.887 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3506.lua
17:57:47.888 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3507.lua
17:57:47.889 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3508.lua
17:57:47.890 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3509.lua
17:57:47.891 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3510.lua
17:57:47.892 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3511.lua
17:57:47.892 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3512.lua
17:57:47.959 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3405.lua
17:57:47.961 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3406.lua
17:57:47.962 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3407.lua
17:57:47.962 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3408.lua
17:57:47.964 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3409.lua
17:57:47.965 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3410.lua
17:57:47.966 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3411.lua
17:57:47.967 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3412.lua
17:57:47.967 <ERROR:ScriptLoader> Could not find script at path Scene/3/scene3_block3413.lua

**Comment by NotThorny** [comment]
These are entirely normal. Those files do not exist so they won't be found. These errors are not related to you not being able to get in.

**Reply by Ting230** [reply]
I reinstalled the game, and now it is version 5.0. Everything else is normal, but after clicking on the game and hoyoerse disappears, it will be stuck on the white screen.
<img width="735" alt="1157bf149b3631a40db9d81a0883663" src="https://github.com/user-attachments/assets/0a378732-4f8d-479f-89ef-091e14ff126e" />
<img width="735" alt="cf9fc64e85e0f6a8c761c1b29280e65" src="https://github.com/user-attachments/assets/d3fc454b-c141-496c-8353-01c355aef6ce" />
<img width="1280" alt="9f6cb38f277de050ac5fae3abc287f8" src="https://github.com/user-attachments/assets/0272c9d7-57ee-4189-8117-0ca28068f5a2" />
<img width="1280" alt="f40bf95f7e8815930888526119ab7c4" src="https://github.com/user-attachments/assets/e895a845-1cbc-4b0c-8cda-c0606d6588b2" />

**Reply by Ting230** [reply, marked as answer]
Thank you very much. I have found the answer on Discord
